src/ia-book-actions.js

```diff
@@ -283,6 +291,38 @@ export default class IABookActions extends LitElement {
         this.autoLoanRenewChecker(true);
       }
     });
+
+    /**
+     * auto-renew interval may stale when tab is in background,
+     * so when user open that tab again,
+     * the [visibilitychange] event can trigger timer to show relavent messages
+     */
+    document.addEventListener('visibilitychange', async () => {
+      if (!document.hidden) {
+        log(
+          'visibilitychange is not hidden time: ------------------ ',
+          new Date().getMinutes(),
+          new Date().getSeconds()
+        );
+
+        if (
+          this.borrowType === 'browsed' &&
+          this.lendingStatus.browsingExpired === false
+        ) {
+          const loanTime = await this.localCache.get(
+            `${this.identifier}-loanTime`
+          );
+          const secondsLeft = Math.round((loanTime - new Date()) / 1000); // different in seconds
```

Comment:
q: what does `different in seconds` comment mean? can we clarify and move to above this var declaration?